src/model/window-guardian.ts

```diff
@@ -175,6 +175,10 @@ export const makeWindowGuardian = (
             cssIDs,
             data: {
                 ...dcrDocumentData,
+                NAV: {
+                    subNavSections: dcrDocumentData.NAV.subNavSections,
+                    currentNavLink: dcrDocumentData.NAV.currentNavLink,
+                },
```

Comment:
I'm probably missing something, but if we no longer hydrate this why do we need this data on `window`?

Reply:
The only probs here are used for the SubNav which is still hydrated